python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -79,23 +90,50 @@ def __init__(self, era, algo = 'csvv2', sfFileName = None, verbose = 0):
                     },
                     'supported_wp' : [ "L", "M", "T", "shape_corr"]
                 },
-                '2017' : {
-                    'inputFileName' : "DeepCSV_94XSF_V4_B_F.csv",
+                '2016FastSim' : {
+                    'inputFileName' : "deepcsv_13TEV_16SL_18_3_2019.csv",
+                    'measurement_types' : {
+                        0 : "fastsim",  # b
+                        1 : "fastsim",  # c
+                        2 : "fastsim"   # light
```

Comment:
What are these? This is expected

Reply:
This is from the csv file that you commented on above and look in the second column, FastSim is what is listed. If by comparison you open the fullSim csv file it has in the second column you have: comb, inc, ect.